modules/network-observability-health-dashboard-overview.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * network_observability/network-observability-operator-monitoring.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-health-dashboard-overview_{context}"]
+= Health dashboards
+
+Metrics about health and resource usage of the Network Observability Operator are located in the *Observe* -> *Dashboards* page in the web console. You can view metrics about the health of the Operator in the following categories:
+
+* *Flows per second*
+* *Sampling*
+* *Errors last minute*
+* *Dropped flows per second*
+* *Flowlogs-pipeline statistics*
+* *Flowlogs-pipleine statistics views*
+** Flows per second
```

Comment:
If these are labels also, I think they should all be bold? (Wasn't sure since I haven't seen this GUI)

Reply:
I was debating what to do with all these sub-panels and I've decided to only document the major name and not document the sub-panels.